app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -2710,20 +2722,28 @@ protected function getProductUrlSuffix($storeId = null)
 
     /**
      * @param array $rowData
-     * @return string
+     * @return string|bool
```

Comment:
see comment before about return type

Reply:
Yes that's a good point, I was on the fence myself about this change.